buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -6,7 +6,7 @@ object Versions {
     const val DD_PLIST = "1.23"
 
     // https://github.com/detekt/detekt
-    const val DETEKT = "1.11.0"
+    const val DETEKT = "1.12.0"
```

Comment:
Hmm I can see that the latest version is `1.14.0`
https://github.com/detekt/detekt/releases

Reply:
we should update Detekt manually in that case, I do not know how `Gradle Dependency Plugin` find new versions

Reply:
Yup, I was going to propose the same. I've also noticed that `1.12` had some issues with `ParametersListWrapping` rule